examples/csr/csr.c

```diff
@@ -166,6 +166,13 @@ int TPM2_CSR_ExampleArgs(void* userCtx, int argc, char *argv[])
         argc--;
     }
 
+    /* init wolfssl, required for fips */
+    rc = wolfSSL_Init();
```

Comment:
This is great, but a couple things.
1) In WOLFCRYPT_ONLY this needs to be `wolfCrypt_Init()`.
2) Use the wolfSSL values `WOLFSSL_SUCCESS`.
3) If you are going to call init you also need to call `_Cleanup()`...